### PR TITLE
fix(ssl): Use shiny new SSL certificate

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: aws/environments/EXAMPLE.yml [hunk 1]
```diff
@@ -3,17 +3,16 @@
 # The following variables should be put into a new yml file in this directory.
 # Any variables used by ansible roles may also be set here.
 
-region: us-west-2
+region: us-east-1
```
Comment (vladikoff) [inline aws/environments/EXAMPLE.yml]: why is this `east` ?

Comment (jbuck) [inline aws/environments/EXAMPLE.yml]: In general, we'd like all AWS resources to be in us-east-1 region by default - I'm hoping people clone this file and use us-east-1 going forward :)

Comment (vladikoff) [inline aws/environments/EXAMPLE.yml]: ohhh, thought we wanted west, sounds good!

**Comment by jrgm** [comment]
There's six new environment configs on the docker branch:
+aws/environments/fxadevtest.yml
+aws/environments/test1.yml
+aws/environments/test2.yml
+aws/environments/test3.yml
+aws/environments/test62.yml
+aws/environments/vbudhram.yml

**Comment by jbuck** [comment]
Good catch!

**Comment by jrgm** [comment]
LGTM

**Pull request merged**